### PR TITLE
Resolve TODO's

### PR DESCRIPTION
**Pull request by djuszczakk**
- TODO 00 - Abandoned
- TODO 01 - Done
- TODO 03 - Done

### CODE REVIEW

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 1]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
 	   - write about uuidsEqual()
  -->
 ## Handling UUIDs and binary data in Bluetooth API
+
+### Hnadling UUIDs
```
Comment (arkadiuszpietraszek) [inline docs/application/web/guides/connectivity/bluetooth.md]: Hnadling -> Handling

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 2]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+Unless stated otherwise, each UUID returned from a function and each UUID attribute of an object may be a 16-bit, 32-bit or 128-bit UUID.
+Bluetooth API functions are case-insensitive with regard to UUIDs - lowercase (abcdef) and uppercase (ABCDEF) characters are recognized as valid hexadecimal digits.
+
+The 128-bit UUID that is the base for UUIDs having 16-bit or 32-bit equivalents is defined available through the API in `BluetoothManager` as `BASE_UUID`.
+
+### Handling binary data
+The `Bytes` type, that aggregates all types in Bluetooth API used to pass binary data. It can be either a *byte[]* or a *DOMString* or a *Uint8Array*.
+
+`BluetoothManager` methods to manipulate binary data:
+   * toByteArray()
```
Comment (arkadiuszpietraszek) [inline docs/application/web/guides/connectivity/bluetooth.md]: I think that all function names outside code examples should be enclosed with `back quote`.

**Review by arkadiuszpietraszek — CHANGES_REQUESTED** [review]
There is a typo in heading.
Besides the typo the content of new section seems technically correct.
I'm not sure though if all the information already included in documentation should be also put here (like describing UUID forms for example), some critical information may be put as a note with the '> **Note**' tag.
I'm also not sure if used writing style is proper to be used in a guide.
I would write using more "guide like" style, for example:
"1. In order to convert an UUID form from ... to ... use the `...` function:
```
[code example here]
```"

**Review by arkadiuszpietraszek — APPROVED** [review]

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 You can use Bluetooth functionalities in your application, such as managing the local Bluetooth adapter, bonding, and exchanging data between Bluetooth-enabled devices. The Bluetooth standard provides a peer-to-peer (P2P) data exchange functionality over short distance between compliant devices.
 
-<!-- TODO #00:
+<!-- TODO #00: features are for application filtering, not related with privileges and not mentioned at most of guides
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md:5]: If features won't be mentioned, I think the comment can be removed.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 4]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toUint8Array(dataInt8Array);
+      var second = tizen.bluetooth.toUint8Array(dataUint8Array);
+      var third = tizen.bluetooth.toUint8Array(dataString);
+      ```
+      Both first, second and third variables are equal.
+
+   * `uuidTo128bit()`
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]: This should go to the section about UUIDs.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 5]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+      var uuidFrom16bits = tizen.bluetooth.uuidTo128bit("1234");
+      var uuidFrom32bits = tizen.bluetooth.uuidTo128bit("ab5690ef");
+      var uuidFrom128bits = tizen.bluetooth.uuidTo128bit("abcdef01-2345-6789-abcd-ef0123456789");
+      ```
+
+      uuidFrom16bits is equal to "00001234-0000-1000-8000-00805f9b34fb"  
+      uuidFrom32bits is equal to "ab5690ef-0000-1000-8000-00805f9b34fb"  
+      uuidFrom128bits is equal to "abcdef01-2345-6789-abcd-ef0123456789"
+
+   * `uuidToShortestPossible()`
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]: This should go to the section about UUIDs.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 6]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+      var from128BitFirst = tizen.bluetooth.uuidToShortestPossible("ab5690ef-0000-1000-8000-00805F9B34FB");
+      var from128BitSecond = tizen.bluetooth.uuidToShortestPossible("abcdef01-2345-6789-abcd-ef0123456789");
+      ```
+
+      from16Bit is equal to "1234"  
+      from32Bit is equal to "acdf"  
+      from128BitFirst is equal to "ab5690ef"  
+      from128BitSecond is equal to "abcdef01-2345-6789-abcd-ef0123456789"
+
+   * `uuidsEqual()`
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]: This should go to the section about UUIDs.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 7]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toByteArray(dataInt8Array);
+      var second = tizen.bluetooth.toByteArray(dataUint8Array);
+      var third = tizen.bluetooth.toByteArray(dataString);
+      ```
+      Both first, second and third variables are equal arrays.
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]: According to https://dictionary.cambridge.org/dictionary/english/both, "both" should be used when referring to 2 things.
```suggestion
      `first`, `second` and `third` variables are equal arrays.
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 8]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toDOMString(dataInt8Array);
+      var second = tizen.bluetooth.toDOMString(dataUint8Array);
+      var third = tizen.bluetooth.toDOMString(dataString);
+      ```
+      Both first, second and third variables are equal to *"0x18b1"*.
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]: According to https://dictionary.cambridge.org/dictionary/english/both, "both" should be used when referring to 2 things.
```suggestion
      `first`, `second` and `third` variables are equal arrays.
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 9]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toUint8Array(dataInt8Array);
+      var second = tizen.bluetooth.toUint8Array(dataUint8Array);
+      var third = tizen.bluetooth.toUint8Array(dataString);
+      ```
+      Both first, second and third variables are equal.
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]: According to https://dictionary.cambridge.org/dictionary/english/both, "both" should be used when referring to 2 things.
```suggestion
      `first`, `second` and `third` variables are equal.
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 10]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+      from128BitSecond is equal to "abcdef01-2345-6789-abcd-ef0123456789"
+
+   * `uuidsEqual()`
+      ```
+      var first = tizen.bluetooth.uuidsEqual("1234", "00001234");
+      var second = tizen.bluetooth.uuidsEqual("ab5690ef", "ab5690ef-0000-1000-8000-00805F9B34FB");
+      var third = tizen.bluetooth.uuidsEqual("abcdef01-2345-6789-abcd-ef0123456789", "abcdef01");
+      ```
+
+      Both first and second variables are true. Variable third is false.
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
      Both `first` and `second` are `true`. `third` is false.
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 11]
```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
...
+
 <!-- END #01 -->
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md:122]: This TODO comment can be removed.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 12]
```diff
@@ -370,7 +460,7 @@ To search for remote Bluetooth devices:
 
 
 <!-- TODO #03:
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md:465]: This TODO comment can be removed.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 13]
```diff
@@ -394,19 +484,20 @@ To connect to other Bluetooth Low Energy devices:
    }
    ```
 
-3. Define a callback for the scan operation that connects to a found device and stops the scan.
+3. Define a callback for the scan operation that connects to a not connected found device and stops the scan.
 
-   Within the callback request, establish a connection with the found device using the `connect()` method of the `BluetoothLEDevice` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothLEDevice), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothLEDevice) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothLEDevice) applications):
+   Within the callback request, check if found device is connected and establish a connection with the not connected device using the `isConnected()` and `connect()` methods of the `BluetoothLEDevice` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothLEDevice), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothLEDevice) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothLEDevice) applications):
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
   Within the callback request, check if the found device is connected and establish a connection with the not connected device using the `isConnected()` and `connect()` methods of the `BluetoothLEDevice` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothLEDevice), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothLEDevice) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothLEDevice) applications):
```

**Review by pwasowski2 — CHANGES_REQUESTED** [review]

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 3 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:5]: All TODO comments will be removed in the last commit.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 11 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:122]: All TODO comments will be removed in the last commit.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 12 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:465]: All TODO comments will be removed in the last commit.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 4 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md]: Done.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 5 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md]: Done.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 6 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md]: Done.

**Review by pwasowski2 — APPROVED** [review]

**Pull request merged**